### ISSUE TITLE
[ACT] HTML page has non-empty title - Inapplicable vs untested

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1022]
Investigate rule differences between ACT inapplicable case and Checker reporting untested for the following:

[HTML page has non-empty title](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/equal-access/#breakdown-2779a5)

[Inapplicable Example 1](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/2779a5/ecc29b73e37b6a125b3fd9767068dcaa368d467a.svg)
**_untested_** - WCAG20_Doc_HasTitle:Pass_0,Fail_1,Fail_2,Fail_3

### Make recommendation:
[ ] Either fix Checker rue, or make recommendation to fix ACT rule and/or testcase
[ ] Open issue 

### Background:
The comparison analysis is documented in a spreadsheet here:
See [Checker_ACT_Mapping 2022-06-23.xlsx](https://ibm.ent.box.com/file/981548094874)
(see column H cells highlighted in yellow for which Checker rules need changing)

Also see the **_Checker Implementation Report_** which documents the rules run against the ACT test cases.
Equal Access Accessibility Checker version 3.1.33-rc, test results were last updated 7 July 2022. 
https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/equal-access/ 

![image](https://user-images.githubusercontent.com/16193609/182186549-e0cc7d4f-bb0c-4990-aeb0-2c0a0e40547c.png)

**Comment by tombrunet** [comment]
svg files are flagged as untested because we don't process them. I'm checking with Wilco about the handling of these.

**Comment by tombrunet** [comment]
Shows as consistent now. Closing.
